website/releases/release-0.14.0.md

```diff
@@ -0,0 +1,319 @@
+---
+title: "Release 0.14.0"
+sidebar_position: 1
+layout: releases
+toc: true
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## [Release 0.14.0](https://github.com/apache/hudi/releases/tag/release-0.14.0) ([docs](/docs/quick-start-guide))
+Apache Hudi 0.14.0 marks a significant milestone with a range of new functionalities and enhancements. 
+These include the introduction of Record Level Index, automatic generation of record keys, the `hudi_table_changes` 
+function for incremental reads, and more. Notably, this release also incorporates support for Spark 3.4. On the Flink 
+front, version 0.14.0 brings several exciting features such as consistent hashing index support, Flink 1.17 support, and U
+pdate and Delete statement support. Additionally, this release upgrades the Hudi table version, prompting users to consult
+the Migration Guide provided below. We encourage users to review the release highlights(TODO Add link),
+[breaking changes](#migration-guide-breaking-changes), and [behavior changes](#migration-guide-behavior-changes) before 
+adopting the 0.14.0 release.
+
+
+
+## Migration Guide
+In version 0.14.0, we've made changes such as the removal of compaction plans from the ".aux" folder and the introduction
+of a new log block version. As part of this release, the table version is updated to version `6`. When running a Hudi job 
+with version 0.14.0 on a table with an older table version, an automatic upgrade process is triggered to bring the table 
+up to version `6`. This upgrade is a one-time occurrence for each Hudi table, as the `hoodie.table.version` is updated in
+the property file upon completion of the upgrade. Additionally, a command-line tool for downgrading has been included, 
+allowing users to move from table version `6` to `5`, or revert from Hudi 0.14.0 to a version prior to 0.14.0. To use this 
+tool, execute it from a 0.14.0 environment (TODO Check what is this tool). For more details, refer to the 
```

Comment:
@lokeshj1703 should be able to help with getting the actual java class or alike for the tool. I am assuming that's what your are asking for?

Reply:
There is support for downgrade via Hudi CLI - https://hudi.apache.org/docs/cli/#upgrade-and-downgrade-table.

Reply:
okay thanks. Will link it here.